src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/WindowsLocalDebugger.xaml

```diff
@@ -42,6 +37,8 @@
         <EnumValue Name="Auto" DisplayName="Auto" Description="Auto" />
         <EnumValue Name="Script" DisplayName="Script" Description="Script" />
         <EnumValue Name="GPUOnly" DisplayName="GPU Only" Description="GPU Only" />
-        <EnumValue Name="ManagedCore" DisplayName="Managed Core" Description="asdf"/>
+        <EnumValue Name="ManagedCore" DisplayName="Managed Core" Description="CoreCLR"/>
+        <EnumValue Name="NativeWithManagedCore" DisplayName="Native with Managed Core" Description="Native and CoreCLR"/>
     </EnumProperty>
+    <StringProperty Name="DotnetPath" Default="C:\Program Files\dotnet\dotnet.exe" Description="Path to the dotnet executable" DisplayName="Dotnet Path"/>
```

Comment:
This should be just dotnet.exe I think so that it picks it up from the path. Otherwise we maybe broken even if it's in the path from a different directory.